domains/fridge.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Fridge is-a.dev",
+  "owner": {
+    "username": "kornls",
+    "email": "larryst.korn@gmail.com"
+  },
+  "record": {
+    "CNAME": "Arq091.github.io"
```

Comment:
We cannot add `fridge.is-a.dev` since the website hasm't been completed.

Reply:
Alright. I will inform you when it is completed. For now, I’ll remove the json file.